pkg/features/kube_features.go

```diff
@@ -1101,13 +1104,13 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	SeparateTaintEvictionController: {Default: true, PreRelease: featuregate.Beta},
 
-	ServiceAccountTokenJTI: {Default: false, PreRelease: featuregate.Alpha},
+	ServiceAccountTokenJTI: {Default: true, PreRelease: featuregate.Beta},
 
-	ServiceAccountTokenPodNodeInfo: {Default: false, PreRelease: featuregate.Alpha},
+	ServiceAccountTokenPodNodeInfo: {Default: true, PreRelease: featuregate.Beta},
 
 	ServiceAccountTokenNodeBinding: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Is there any value in promoting this to beta but leaving it off by default?

Reply:
not sure - it might be a bit confusing for an admin that _expects_ beta features to be enabled by default.. OTOH, it is a bit confusing as we *do* consider this feature beta in terms of stability - just we don't want it *enabled* by default...

@liggitt any thoughts?